### PR TITLE
Refine Status API result

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes
This PR refines the stateMigrationStatus result.

- Add total read count
- fixed progress percent value
- fixed total committed count
- fixed err string

```
> admin.stateMigrationStatus
{
  committed: 26994,
  err: "null",
  isMigration: true,
  migrationBlockNumber: 9208286,
  pending: 15886,
  progress: 0.48828125,
  read: 28945
}
```

```
> admin.stateMigrationStatus
{
  committed: 60817,
  err: "stop state migration",
  isMigration: false,
  migrationBlockNumber: 9175529,
  pending: 15693,
  progress: 1.171875,
  read: 62737
}
```

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [ ] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: blockchain/state_migration.go [hunk 1]
```diff
@@ -376,8 +375,8 @@ func (bc *BlockChain) StopStateMigration() error {
 
 // StateMigrationStatus returns if it is in migration, the block number of in migration,
 // number of committed blocks and number of pending blocks
-func (bc *BlockChain) StateMigrationStatus() (bool, uint64, int, int, float64, error) {
-	return bc.db.InMigration(), bc.db.MigrationBlockNumber(), bc.committedCnt, bc.pendingCnt, bc.progress, bc.migrationErr
+func (bc *BlockChain) StateMigrationStatus() (bool, uint64, int, int, int, float64, error) {
```
Comment (ehnuje) [inline blockchain/state_migration.go:378]: There are too many return variables. How about declaring a struct for state migration status?

Comment (KimKyungup) [reply blockchain/state_migration.go:378]: How about this function returns `map[string]interface{}`?
If it returns a struct, we need to declare the struct type.
But I can't find good place where I define the struct type.
Any good place?

File: blockchain/blockchain.go [hunk 2]
```diff
@@ -165,6 +165,7 @@ type BlockChain struct {
 	// State migration
 	prepareStateMigration bool
 	stopStateMigration    chan struct{}
+	readCnt               int
 	committedCnt          int
 	pendingCnt            int
 	progress              float64
```
Comment (winnie-byun) [inline blockchain/blockchain.go:171]: The variable names are not quite accurate as members in `Blockchain`.
This suggestion might not be the right place to write in this PR. However, what do you think about defining a new struct responsible for state migration?
```
types Blockchain struct {
...
  stateMigration stateMigration
...
}
```
```
type stateMigration struct {
	isPrepare  bool
	stopSignal chan struct{}
	Err        error

	migrationStats
}
```

Comment (KimKyungup) [reply blockchain/blockchain.go:171]: It's good idea. But I think modifying now is not good for this stage.

**Comment by KimKyungup** [comment]
@winnie-byun @ehnuje I think modifying much is not good for now.
After 1.5.0 release, I will refactor this code as you suggested.
For now, let's go like this.

File: blockchain/state_migration.go (hunk 1 above)
Comment (ehnuje) [reply blockchain/state_migration.go:378]: If API is the only place where `StateMigrationStatus` is used, I'm okay with returning the result as as a map.

**Pull request merged**